backend/app/auth.py

```diff
@@ -92,13 +99,20 @@ def login():
         resp.set_cookie("auth_jwt", token)
         return resp
 
-
+    return Response(status=400, response="Forbidden")
```

Comment:
I know the messages themselves are misleading but 401 is for authentication and 403 is for authorization. ([Reference](https://www.rfc-editor.org/rfc/rfc9110.html#name-401-unauthorized))

So for line 84 it should be 401 and "Forbidden" line 102 is somehow ambiguous.

But I'm fine to leave them this way.